server/src/command_replay_create.go

```diff
@@ -45,7 +45,12 @@ func commandReplayCreate(ctx context.Context, s *Session, d *CommandData) {
 	} else if d.Source == "json" {
 		// Before creating a new game and emulating the actions,
 		// ensure that the submitted JSON does not have any obvious errors
-		if valid, _ := isJSONValid(s, d); !valid {
+
+		if d.GameJSON == nil {
```

Comment:
this should be part of the isJSONValid function

Reply:
it will break command_table_create.go:L91:96

Reply:
oh I see